PiWriteBack/Config.cs

```diff
@@ -1,20 +1,34 @@
 ﻿using Cognite.Extractor.Utils;
 using System.Collections.Generic;
 
-namespace PiWriteBack
+namespace Cognite.PiWriteBack
 {
-    public class Config : BaseConfig
+    public class FullConfig : BaseConfig
     {
         public PiConfig Pi { get; set; }
         public TimeSeriesConfig TimeSeries { get; set; }
         public int PollingInterval { get; set; } = 60_000;
```

Comment:
More of a QoL thing, but have a look at https://github.com/cognitedata/dotnet-extractor-utils/blob/master/Cognite.Common/TimeSpanWrapper.cs

It's a utility intended to let you accept options like `60s`, or `5m` in place of just a milliseconds integer. Intended for things like this. You can even use the `CronTimeSpanWrapper`, which also accepts a cron-expression.